pkg/services/vimmachine_test.go

```diff
@@ -237,3 +237,59 @@ var _ = Describe("VimMachineService_GetHostInfo", func() {
 	})
 
 })
+
+var _ = Describe("VimMachineService_createOrPatchVSPhereVM", func() {
```

Comment:
if you have more use cases in mind, happy to increase the coverage in a follow up.

Reply:
Sure this is a good enough place to get started